amberdata/test/balance.test.ts

```diff
@@ -0,0 +1,230 @@
+import { assert } from 'chai'
```

Comment:
Let's try to reuse these tests!

The idea is to define a specific behavior and reuse it in the individual tests.
For inspiration take a look at how [OpenZepplin structures its tests and behaviors on the ERC20 contract example.](https://github.com/OpenZeppelin/openzeppelin-contracts/tree/release-v3.2.0/test/token/ERC20)